packages/integrations/react/src/index.ts

```diff
@@ -7,6 +7,7 @@ export type ReactIntegrationOptions = Pick<ViteReactPluginOptions, 'include' | '
 	experimentalReactChildren?: boolean;
 };
 
+// @ts-expect-error
```

Comment:
The React Vite plugin only exports itself as a default export with a lying type to TypeScript's eye, so this doesn't work. It'd be easier if it exported itself normally, like the Preact plugin does.